### PR TITLE
Upgrade Rails to 4.1.14.1

### PR DESCRIPTION
**Pull request by elliotcm**
http://weblog.rubyonrails.org/2016/1/25/Rails-5-0-0-beta1-1-4-2-5-1-4-1-14-1-3-2-22-1-and-rails-html-sanitizer-1-0-3-have-been-released/

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -154,3 +156,6 @@ DEPENDENCIES
   simplecov-rcov (= 0.2.3)
   spring
   unicorn (= 4.8.2)
+
+BUNDLED WITH
+   1.11.2
```
Comment (benlovell) [inline Gemfile.lock]: This may cause problems.

Comment (elliotcm) [inline Gemfile.lock]: Removed.

**Comment by benlovell** [comment]
```
Default string limit for postgres is not set to 255.  Review monkey-patch in /home/jenkins/workspace/govuk_content-register_branches/config/initializers/remove_default_column_limit.rb.
```

**Comment by elliotcm** [comment]
Thanks @benlovell, we were looking into it.  Fixed now.

**Pull request merged**